editor/export/editor_export_preset.h

```diff
@@ -153,6 +153,8 @@ class EditorExportPreset : public RefCounted {
 	String get_script_encryption_key() const;
 
 	Variant get_or_env(const StringName &p_name, const String &p_env_var, bool *r_valid = nullptr) const;
+	// Return version number, or fall back to the `application/config/version` project setting if set to an empty string.
```

Comment:
I don't see the implementation of that fallback.

Reply:
I believe this is the fallback: https://github.com/Calinou/godot/blob/281f988bdb31f7c542fbf363622459f2ca91298c/editor/export/editor_export_preset.cpp#L338-L340

Reply:
Right, that part is correct.